### PR TITLE
WL-3792 Adding warning message that files will not be overwritten

### PR DESCRIPTION
**Pull request by surya-janani**

### CODE REVIEW

**Comment by nicholaswilson100** [comment]
Did you manage to do everything in the Description in the Jira? e.g., check that it happens, discuss where the best place to put the text is, etc.

**Comment by nicholaswilson100** [comment]
The property value now includes text that is mentioned in another property:

label.version        = Upload a new version

Is there any way we could re-use this so that if someone wants to change the text 'Upload a new version' they don't have to change it in 2 places.

Maybe look at how this one works?

paste.error     = Could not add ''{0}'' to this folder.

**Comment by buckett** [comment]
If you push new code to a branch that is part of a pull request and you want somone to review it you need to comment on the pull request to say so.

File: content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesHelperAction.java [hunk 1]
```diff
@@ -171,6 +171,8 @@
 	/** Tool property to enable Drag and Drop uploads in a per-tool basis */
 	private static final String TOOL_PROP_DRAGNDROP_ENABLED = "content.upload.dragndrop";
 
+	private static final String LABEL_VERSION = "label.version";
```
Comment (buckett) [inline content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesHelperAction.java]: When you're only using this string in one place it isn't worth extracting out a constant for it. The class becomes more cluttered as a result. Sorry for not spotting this the first time around.

Comment (surya-janani) [inline content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesHelperAction.java]: This was made dynamic with the thought that, if in future {label.version} is worded differently ,then it requires changing in 2 places manually. 

Do you think its not worth it.?

Comment (buckett) [inline content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesHelperAction.java]: Creating a constant for a string that is only use in one place and is only used as a i18n lookup key makes the code harder to follow. It also doesn't fit with the existing pattern that there is in this file (no i18n lookup keys are constants).

**Comment by surya-janani** [comment]
Din't notice Nick's comments . Making changes to not reuse label.version

**Comment by surya-janani** [comment]
Misunderstood comments.

**Comment by buckett** [comment]
You need to make a comment on the pull request after pushing code.

Also think about variable names, having the Java variable names different to the resource bundle string and the velocity context names makes the code harder to read.

**Pull request merged**